### DISCUSSION TITLE
just some thoughts :)

### DISCUSSION THREAD

**Discussion by ghost** [Slow-Scallion4183/Adventure-Colussus#28] [Ideas]
we can use rich for colors and ascii for characters. im asking colors because it would be nice and can we do our game like 3d? cuz it would be nice!

**Comment by Slow-Scallion4183** [comment]
we are using 2d which means we can still use the rich library for colouring and high-lighting text and so on. i dont think it even works for 3d but if it did its not the kind of approach we are coming at when making the game.

**Reply by Slow-Scallion4183** [reply]
but yeah still good idea! keep them coming!

**Reply by ghost** [reply]
ok. i just bothered out the chests cuz they block our way lol.

**Reply by Slow-Scallion4183** [reply]
not exactly. we can choose to move out of their way
